src/components/input/range/index.tsx

```diff
@@ -18,19 +19,22 @@ export class SmoothlyInputRange {
 		this.smoothlyInput.emit({ [this.name]: this.value })
 	}
 	inputHandler(e: Event): void {
```

Comment:
missed this last time. please write out the entire variable name. we prefer to not use acronyms. I believe it is ok to use acronyms in callbacks but here we should write it out.